tests/ui/crashes/ice-360.stderr

```diff
@@ -14,10 +14,17 @@ error: this loop could be written as a `while let` loop
 LL | /     loop {
 ...  |
 LL | |     }
-   | |_____^ help: try: `while let Some(ele) = iter.next() { .. }`
+   | |_____^
    |
    = note: `-D clippy::while-let-loop` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::while_let_loop)]`
+help: try
+   |
+LL ~     while let Some(ele) = iter.next() {
+LL +         let _ = ele;
```

Comment:
Should probably also ignore these as they're no-ops (doesn't even read from `ele`).

Reply:
Done.